plugins/custom-class/prism-custom-class.js

```diff
@@ -7,24 +7,61 @@ if (
 	return;
 }
 
+/**
+ * @callback ClassMapper
+ * @param {string} className
+ * @param {string} language
+ * @returns {string}
+ */
+/**
+ * @typedef CustomClassOptions
+ * @property {ClassMapper} classMap
+ * @property {string} prefixString
```

Comment:
I don't think this `typedef` is right: `classMap` can be either a function or an object, right?

Reply:
The `classMap` of the `map` function can be [either function or object](https://github.com/RunDevelopment/prism/blob/b2cfbd1afcad8468c5a5a3d85b01cc2f92ce26f5/plugins/custom-class/prism-custom-class.js#L37) but this is the type definition of the object storing the [internal state](https://github.com/RunDevelopment/prism/blob/b2cfbd1afcad8468c5a5a3d85b01cc2f92ce26f5/plugins/custom-class/prism-custom-class.js#L26) of the plugin.
`map` will [convert its given parameter into a function](https://github.com/RunDevelopment/prism/blob/b2cfbd1afcad8468c5a5a3d85b01cc2f92ce26f5/plugins/custom-class/prism-custom-class.js#L43), so the type definition is correct.